_includes/unscheduled-purchase.md

```diff
@@ -22,12 +22,12 @@ Content-Type: application/json
 {
     "payment": {
         "operation": "UnscheduledPurchase",
-        "intent": "Authorization",
+        "intent": "Authorization|AutoCapture",
```

Comment:
No pipes in code examples, please. List the possible values for a given property in a property table below the example instead.

```suggestion
        "intent": "Authorization",
```

Reply:
I figured you'd comment on those (ref. my initial comment :) )